ethereumj-core/src/main/java/org/ethereum/manager/BlockLoader.java

```diff
@@ -112,7 +112,7 @@ public void loadBlocks() {
                 Path path = Paths.get(fileSrc);
                 // NOT OPTIMAL, but fine for tests
                 byte[] data = Files.readAllBytes(path);
-                RLPList list = RLP.decode2(data);
+                RLPList list = RLP.decode2ListNarrow(data);
```

Comment:
It turns that this update changes the logic: `RLP.decode2(data, Integer.MAX)` is replaced with `RLP.decode2(data, 2).get(0)`. Additional `get(0)` is added in the latter statement.

Reply:
Yeah, I've checked Hive benchmarks this method was added for, they use this kind of list wrapping. Fixed